docs/organizations/managing-security-and-risk.md

```diff
@@ -34,7 +34,10 @@ The Security and Risk Management item list displays a list of items, filtered by
 
 To access the item list, access the [dashboard](#dashboard) and click the **Review** button under the area of interest, based on the desired filtering.
 
-To find out more about an item, click its **Details** column. This redirects you to a filtered view of the **Repository issues** page, showing only the item of interest.
+When viewing the item list:
+
+-   To update the filtering criteria, click the **Status** dropdown or the **Priority** dropdown above the list.
```

Comment:
Possible simplification to avoid repeating "[drop-down](https://grammarhow.com/dropdown-drop-down-or-drop-down/)". :wink: 

```suggestion
-   To update the filtering criteria, click the **Status** or **Priority** drop-downs above the list.
```